core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -222,8 +220,6 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
      * @notice After a passed withdrawal request (i.e., by a call to `requestWithdrawal` and waiting
      * `withdrawalLiveness`), withdraws `positionData.withdrawalRequestAmount` of collateral currency.
      */
-    // TODO: is onlyCollateralizedPosition(msg.sender) correct here? if a position withdraws all their collateral will this still work?
```

Comment:
@chrismaree how would you replicate this in a test?

Reply:
FWIW: I believe that `onlyCollateralizedPosition` does work as intended: checking if collateral > 0

Reply:
ye, I agree